### ISSUE TITLE
Getting "No default export found in module  import/default" running eslint

### ISSUE THREAD

**Issue by jacargentina** [react-native-community/react-native-share#507]
I'm getting this when trying to lint a file in my RN project.

![eslint](https://user-images.githubusercontent.com/517228/58434469-31c3b200-8092-11e9-9754-69ce9fb0aec5.png)

Current exports for the library:

![current](https://user-images.githubusercontent.com/517228/58434578-c75f4180-8092-11e9-934e-29767bba64a7.png)

should be converted to proper ES6 syntax https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Sentencias/export

**Comment by jgcmarins** [comment]
Maybe just refactoring from this: 
```tsx
module.exports = RNShare;
module.exports.Overlay = Overlay;
module.exports.Sheet = Sheet;
module.exports.Button = Button;
module.exports.ShareSheet = ShareSheet;
```

Into this:
```tsx
export { Overlay, Sheet, Button, ShareSheet };
export default RNShare;
```

**Comment by jgcmarins** [comment]
https://github.com/react-native-community/react-native-share/pull/509

**Comment by MateusAndrade** [comment]
> #509

I will give a look at this PR/ISSUE on this issue. Btw, do you think changing module.exports to export default would be a better idea?

**Comment by jgcmarins** [comment]
yes, looks like it's a best practice

**Comment by MateusAndrade** [comment]
@jacargentina just merged #509
